### ISSUE TITLE
Unable to Share image on fullscreen view 

### ISSUE THREAD

**Issue by Mrraffikki** [ccrama/Slide#467]
If I try to share an image via the < (Share Icon on bottom right of screen), pop up comes up "Uh oh, something went wrong!/Would you like to try again ("NO" "YES"), "NO"  removes the pop up, "YES" returns the same pop up.

(Also, I had to manually allow storage in Settings/Apps/Slide to be able to download, and restart Slide. Possibly this issue requires Contacts, which is not allowed currently on my phone?)

TL;DR: I believe sharing and downloading needs Contacts and Storage permissions, which currently are needed to be manually accepted via settings, not usual Marshmallow setting pop up that comes in.

I'm on Nexus 5x running 6.0, build MDB08I

**Comment by Alexendoo** [comment]
This is because the file is written to storage, then shared as an image file rather than a link I believe. Related: #475

**Comment by l3d00m** [comment]
This is fixed in the next update!
